src/schema.rs

```diff
@@ -0,0 +1,75 @@
+//! Schema
+
+#![allow(missing_docs)]
+use template;
+use std::collections::BTreeMap;
+use serde_json::Value;
+
+/// Schema
+pub struct Schema {
+    files: BTreeMap<String, File>,
+}
+
+/// Supported output file formats
+pub enum FileFormat {
```

Comment:
Should we have some logic to transform a string to a `FileFormat`, given we'll need to do that transformation when parsing the schema.